tools/lsmcli/cmdline.py

```diff
@@ -798,6 +859,12 @@ def _get_item(l, the_id, friendly_name='item', raise_error=True):
     ['vilc', 'volume-ident-led-clear'],
     ['srcpu', 'system-read-cache-pct-update'],
     ['pmi', 'pool-member-info'],
+    ['ldl', 'local-disk-list'],
+    ['lb', 'list --type batteries'],
+    ['vci', 'volume-cache-info'],
+    ['vcp', 'volume-phy-disk-cache-set'],
```

Comment:
The reason to introduce lsmcli alias is to save key typing[1] and easy to remember.
Personally, `vcp` is easy to type and memorize than `vpdcs` or future `vpdcu`.

Just FYI, using first letter of each word might cause problem for `volume-delete` and `volume-disable`.

[1]: I use lsmcli a lot during coding. So the lsmcli alias do save a lot of my time.
Anyway, I could live with both ways for this trivial issue. @tasleson, any input?